spec/sec-regexp-abstract-operations-patch.html

```diff
@@ -34,7 +34,7 @@ <h1><ins>FromMatchOptions ( _Options_ )</ins></h1>
     1. Assert: _Options_ is a MatchOptions Record.
     1. Let _obj_ be ObjectCreate(%ObjectPrototype%).
     1. Assert: _obj_ is an extensible ordinary object with no own properties.
-    1. Perform CreateDataProperty(_obj_, `"capture"`, _Options_.[[Capture]]).
+    1. Perform ! CreateDataProperty(_obj_, `"capture"`, _Options_.[[Capture]]).
```

Comment:
Is it better to do `! CreateDataProperty` and then later Assert that it returns *true*, or do `! CreateDataPropertyOrThrow`, which basically combines both?

Reply:
The latter seems much better, good call!

Reply:
I've seen both used that way in the spec, so there may be places calling `! CreateDataProperty` and not checking for `true` currently.